linera-service/tests/end_to_end_tests.rs

```diff
@@ -1317,14 +1317,14 @@ async fn test_resolve_binary() {
     );
 }
 
+//#[cfg_attr(feature = "scylladb", test_case(LocalNetConfig::new_test(Database::ScyllaDb, Network::Udp) ; "scylladb_udp"))]
```

Comment:
This is acceptable since we don't use UDP in production but I would create a task and add a `TODO(#num): investigate test failure with ScyllaDb + udp`

Reply:
It may belong to the other PR containing the pagination changes to ScyllaDb, though.

Reply:
No, because those bugs were showing up already in that branch with the pagination changes.